config/initializers/seeds_post_finalize_plugins.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.config.after_initialize do
+  Foreman.settings.load_values unless Foreman.in_setup_db_rake? || !(Setting.table_exists? rescue false)
+
+  Foreman::Plugin.registered_plugins.each do |_name, plugin|
+    plugin.finalize_setup!
+  end
```

Comment:
Good point, extracted only the part we needed, moved into `y_finalize_plugins` so that I don't have to touch the `z_print_slow_initializers` while still letting it run last